### PR TITLE
added the team page

### PR DESCRIPTION
**Pull request by ArshanKhanifar**
hey guys this is the team page, still needs to become responsive and right now there's a bunch of dummy text and dummy photos used... but the base work is done and it's ready...

### CODE REVIEW

**Comment by tituschow** [comment]
Just a heads up, I'm not involved with website development.

File: team.html [hunk 1]
```diff
@@ -15,24 +15,166 @@
...
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"/>
-    <script src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
-    <!-- <script src="js/bundle.js"></script> -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+
+
+    <script src="https://d3js.org/d3.v4.min.js"></script>
+    <script src="js/loader.js"></script>
+    <script src="js/team.js"></script>
+
```
Comment (STBTony) [inline team.html:198]: need a `<script src="js/header.js"></script>` here @ArshanKhanifar

**Comment by STBTony** [comment]
@ArshanKhanifar `img/team-cover.jpg → img/team/electrical-team.jpg` what's this!? I'm using the same picture in `index.html`

**Comment by STBTony** [comment]
@ArshanKhanifar also for some reason the headers are not working as intended on team page

**Comment by STBTony** [comment]
@ArshanKhanifar hey arshan just a heads up, team page doesn't have a preloader yet, and the headers are kinda messed up, but don't fix it yet cuz i'm updating your base branch

**Comment by STBTony** [comment]
@ArshanKhanifar when you scroll down the pagination goes over the midsun logo, too high z-index?

**Comment by STBTony** [comment]
@ArshanKhanifar some margin bottom at the end of the page would probably be a good idea

**Pull request closed without merging**